### PR TITLE
blog: Experience as an LFX Mentee for Chaos Mesh

### PR DESCRIPTION
**Pull request by TangliziGit**
Here is the blog about my LFX mentee experience. But I am not sure what the `slug` should be in the front matter.

Signed-off-by: TangliziGit <tanglizimail@foxmail.com>

### CODE REVIEW

File: blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Most of these metrics are relatively simple to collect, requiring only simple queries to database objects, k8s objects, or some simple counts. However, there are some special metrics that are difficult to collect. For example, you need to query the data by executing commands in the network namespace of the corresponding container, or query all the containers under the daemon through three different container runtimes, or collect data on the communication between the gRPC client and the server.
+
+These tasks were strange to me. Therefore, I had to ask my mentor for technical support from time to time, and he was always very responsive. I was greatly impressed by my mentor’s extensive knowledge and experience in this field. Under the guidance from my mentor, I was finally able to put together the [RFC](https://github.com/chaos-mesh/rfcs/pull/23) document for my design. Later, in order to track my work, I created a [tracking issue](https://github.com/chaos-mesh/chaos-mesh/issues/2397).
+
+![Tracking issue](/static/image/2022-01-14/lfx-mentee-experience-tracking-issue.png)
```
Comment (STRRL) [inline blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md:34]: this URL needs updates
```suggestion
![Tracking issue](/img/lfx-mentee-experience-tracking-issue.png)
```

Comment (STRRL) [reply blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md:34]: other images also need update :)

Comment (TangliziGit) [reply blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md:34]: OK

File: blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+slug:
```
[the remark this answers is not in the document]
Comment (TangliziGit) [reply blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md]: OK :-)

File: blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+author_url: https://github.com/TangliziGit
+author_image_url: https://avatars.githubusercontent.com/u/40566218?v=4
+image: /img/lfx-mentee-experience-banner.png
+tags: [Chaos Mesh, Chaos Engineering, LFX Mentorship, Monitoring Metrics]
+---
+
+![Experience as an LFX Mentee for Chaos Mesh](/img/lfx-mentee-experience-banner.png)
+
+I am a graduate student studying software engineering at Nanjing University. My research focuses on DevOps, which has intrinsic connections with chaos engineering and observability. To get involved in the open-source community, understand Kubernetes more deeply, and experience the daily jobs around infrastructure, I applied for the CNCF LFX Mentorship in Fall 2021 to work on the [Chaos Mesh](https://github.com/chaos-mesh/chaos-mesh) project.
+
```
[the remark this answers is not in the document]
Comment (TangliziGit) [reply blog/2022-01-14-experience-as-an-lfx-mentee-for-chaos-mesh.md:15]: Done.

**Pull request merged**